pkg/agent/flowexporter/exporter/exporter_perf_test.go

```diff
@@ -0,0 +1,269 @@
+// +build !race
+
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exporter
+
+import (
+	"flag"
+	"fmt"
+	"math/rand"
+	"net"
+	goruntime "runtime"
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	"k8s.io/klog"
+
+	"antrea.io/antrea/pkg/agent/flowexporter"
+	"antrea.io/antrea/pkg/agent/flowexporter/connections"
+	connectionstest "antrea.io/antrea/pkg/agent/flowexporter/connections/testing"
+	"antrea.io/antrea/pkg/agent/flowexporter/flowrecords"
+	interfacestoretest "antrea.io/antrea/pkg/agent/interfacestore/testing"
+	proxytest "antrea.io/antrea/pkg/agent/proxy/testing"
+	"antrea.io/antrea/pkg/ipfix"
+	queriertest "antrea.io/antrea/pkg/querier/testing"
+)
+
+const (
+	testPollInterval           = 2 * time.Second
+	testConnectionsCount       = 30000
```

Comment:
Looking at the other counts, it seems like you consider 9K connections to stay the same (30K - (20K +1K)). I feel they need to be updated (at least times) or die.